src/silx/math/_colormap.pyx

```diff
@@ -101,7 +106,7 @@ ctypedef fused image_types:
 
 # Normalization
 
-ctypedef double (*NormalizationFunction)(double) nogil
+# ctypedef double (*NormalizationFunction)(double) nogil
```

Comment:
This is no longer a valid syntax?

Reply:
It is still valid but it is unused ... so I commented it to ensure nothing breaks.
To be removed once we are sure it makes no regression anywhere.